e2e_samples/parking_sensors/scripts/deploy_appservice.sh

```diff
@@ -0,0 +1,27 @@
+export appName=data-simulator-$DEPLOYMENT_ID
+
+arm_output=$(
```

Comment:
I would like this to be consistent with the rest of the code. The bicep piece should be consolidated on the main.bicep file like the other modules.